release/rebuild_bom_files.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+source _bom.sh
+source _git.sh
+source _hotfix.sh
+source _jdk.sh
+source _liferay_common.sh
+source _package.sh
+source _patcher.sh
+source _product.sh
+source _publishing.sh
+
+	#
+	# TODO Remove once all systems are using LIFERAY_RELEASE_GIT_REF instead of LIFERAY_RELEASE_GIT_SHA
+	#
+function check_usage {
+	if [ -z "${LIFERAY_RELEASE_GIT_REF}" ]
+	then
+		print_help
+	fi
+
+	if [ -z "${LIFERAY_RELEASE_PRODUCT_NAME}" ]
+	then
+		LIFERAY_RELEASE_PRODUCT_NAME=dxp
+	fi
+
+	_BUILD_TIMESTAMP=$(date +%s)
+
+	_RELEASE_TOOL_DIR=$(dirname "$(readlink /proc/$$/fd/255 2>/dev/null)")
+
+	lc_cd "${_RELEASE_TOOL_DIR}"
+
+	mkdir -p release-data
+
+	lc_cd release-data
+
+	_RELEASE_ROOT_DIR="${PWD}"
+
+	_BUILD_DIR="${_RELEASE_ROOT_DIR}"/build
+	_BUILDER_SHA=$(git rev-parse HEAD)
+	_BUNDLES_DIR="${_RELEASE_ROOT_DIR}"/dev/projects/bundles
+	_PROJECTS_DIR="${_RELEASE_ROOT_DIR}"/dev/projects
+	_RELEASES_DIR="${_RELEASE_ROOT_DIR}"/releases
+	_TEST_RELEASE_DIR="${_RELEASE_ROOT_DIR}"/test_release
```

Comment:
@pgydev, do you really need all these variables?

Reply:
@pgydev  yes we need them, except the `_TEST_RELEASE_DIR`